test/modules/linear_boltzmann_solvers/transport_keigen/c5g7/c5g7_jfnk.lua

```diff
@@ -74,6 +83,32 @@ elseif (k_method == "pi_scdsa_pwlc") then
     })
     solver.Initialize(k_solver)
     solver.Execute(k_solver)
+    elseif (k_method == "pi_smm") then
+        k_solver = lbs.PowerIterationKEigenSMM.Create
+        ({
+            lbs_solver_handle = phys1,
+            accel_pi_verbose = true,
+            k_tol = 1.0e-8,
+            accel_pi_k_tol = 1.0e-8,
+            accel_pi_max_its = 30,
+            diff_sdm = "pwlc",
+        })
+        solver.Initialize(k_solver)
+        solver.Execute(k_solver)
+    elseif (k_method == "pi_smm_pwld") then
+        k_solver = lbs.PowerIterationKEigenSMM.Create
+        ({
+            lbs_solver_handle = phys1,
+            accel_pi_verbose = true,
+            k_tol = 1.0e-8,
+            accel_pi_k_tol = 1.0e-8,
+            accel_pi_max_its = 30,
+            diff_sdm = "pwld"
+        })
+        solver.Initialize(k_solver)
+        solver.Execute(k_solver)
+        solver.Initialize(k_solver)
+        solver.Execute(k_solver)
```

Comment:
same as before. in addition, why are the instructions repeated here?

Reply:
Typo. Good catch.